ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/GeoCityLookup.java

```diff
@@ -123,7 +124,8 @@ public PubsubMessage apply(PubsubMessage message) {
           CityResponse response = geoIP2City.city(ipAddress);
           foundCity.inc();
 
-          attributes.put("geo_country", response.getCountry().getIsoCode());
+          String countryCode = response.getCountry().getIsoCode();
+          attributes.put("geo_country", Normalize.countryCode(countryCode));
```

Comment:
so, we can't just trust the paid maxmind geoip database to be giving us valid/normalized country codes? 😱 😨

Reply:
You make a very good point, and I don't have any reason to believe maxmind won't give us valid codes. The normalization here is basically just an insurance policy and to keep conceptually in line with other the other `normalized_` fields we're going to be adding. I'll likely take the normalize call out of GeoCityLookup as part of the larger metadata refactor that's coming.